### PR TITLE
Host disconnect

### PR DESCRIPTION
**Pull request by ocjarman**
working :)

### CODE REVIEW

File: src/store/gameSlice.ts [hunk 1]
```diff
@@ -8,7 +8,8 @@ interface initialStateType {
   guessesRemaining: number;
   winner: string;
   loser: string;
-  gameHistory: []
+  gameHistory: [];
+  host: {} | null;
```
Comment (Joshowens16) [inline src/store/gameSlice.ts:12]: (referring to the 'host' type): Unlike the last comment, I think this one needs to be fixed in this merge, the last one Rose and I fixed on the front end branch.
So here host can either be a boolean or null. `host: boolean | null` or its just a boolean and we init the state as false? Up to you!

File: src/store/gameSlice.ts [hunk 2]
```diff
@@ -20,6 +21,7 @@ const initialState: initialStateType = {
   winner: '',
   loser: '',
   gameHistory: [],
+  host: null,
```
Comment (Joshowens16) [inline src/store/gameSlice.ts:24]: here you can see when I first go to the page, host is just set to false, so maybe the init the key as false? If so then the type would only need to be boolean
Screen Shot 2023-02-03 at 8 59 08 AM

**Review by Joshowens16 — CHANGES_REQUESTED** [review]
Just a that one issue with the type for isHost and whether or not isHost should be initiated as false in!

File: src/store/gameSlice.ts [hunk 3]
```diff
@@ -8,7 +8,8 @@ interface initialStateType {
   guessesRemaining: number;
   winner: string;
   loser: string;
-  gameHistory: []
+  gameHistory: [];
```
[the remark this answers is not in the document]
Comment (ocjarman) [reply src/store/gameSlice.ts:11]: yes!! i see this in my current typescript branch and it makes much more sense how you're describing

File: src/store/gameSlice.ts (hunk 2 above)
Comment (ocjarman) [reply src/store/gameSlice.ts:24]: i think you are mixing up 'host' and 'isHost' ?

**Review by Joshowens16 — APPROVED** [review]

**Pull request merged**